### ISSUE TITLE
Document "_config" 

### ISSUE THREAD

**Issue by randallmeeker** [balderdashy/sails-docs#410]
The documentation makes reference to the "_config" option to outline blueprint properties. But I know I have come across other features not listed in blueprint properties such as defining the Model or default query params. 

I know I'm curious as to what all it can do. I think setting a bunch of this stuff on the controller is better than placing it in other places (like creating policies.) It seems to be answering quite a few questions on SO as well.

**Comment by sgress454** [comment]
Ok!  Finally got a chance to dig into this, and I think it's safe to say the following:

1. `_config` will override default blueprint config for a controller
1. `_config` can be used to provide default route options for blueprint routes

Those should probably be two different things.  Things like `shortcuts: true` have no place in the route options, but the blueprint code has evolved in such a way that everything is just merged into the `routeOpts` before the shadow routes are bound.  Be that as it may, this should be documented, probably under the "Controllers" section in "Concepts".  Ideally we'd deprecate the use of `_config` to override route params in the next release (offering a different key instead), and then eliminate it the release after that.

**Comment by randallmeeker** [comment]
Ya, that blueprint config for the controller is powerfull and I like it, but it is hard to translate it from the docs because the docs reference it in terms of a route (or controller.action). It makes since, but can be confusing.

So are docs in order? or wait to see if its going to be depreciated and written differently? 

I think this config key could be really expanded. Direction on where we think this is headed would be cool because I might take a crack at adding functionality to it. At least I am more comfortable putting policies, blueprint config everything in the controller because I think it makes it more modular and keeps everything in the same place vs skipping around.

**Comment by sailsbot** [comment]
Thanks for posting, @randallmeeker.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
